.travis.yml

```diff
@@ -12,5 +12,15 @@ notifications:
   email: false
 
 script:
-  - yarn lint
-  - yarn danger ci
+- yarn lint
+- yarn danger ci
+
+deploy:
+  provider: npm
+  email: xander.marjoram@just-eat.com
+  api_key:
+    secure: ELvuVI5RvAH4p7xz6VcaM8ibHgiMOect+UK/iE2W96qJwZQXOKM9nIKXXIejclICPdT/noOCvQlNNxet89HynZrv/C4rgxGlcRHdILwf5J2KU1Qyu+Me/S4tpbl70jU5y14RB327j0JsnD9liqst11h7m4qJ7UTsNHZRADWgMVYrwt8aJAkONDS+lhDFilNEB7rWSRGnk66d1AmTAFNxUTiaf78G+xMTQo0zX10HgsWLGuW63DLV9A4Bo+MNXw40j0mSVGctXfBuZ3NvW7+Gnk9HI8jaVZVtfAVHfxQfGr2X+HPTv5/VaPaztiOOT2TswuJhV2Zybe+VbfobL0tG3NigL42ldYao8UaNHrozLxCkLloXZJzbr5vhv/9CiZGd1awJ9+bI8JiRidMwcvsRqrak3YP8e6OPBB/ePcm5bL6TDykyDIhYA7pJTFyTaovbnac3h7wYx6881suj8AY4UYThUcNCqyjiVxfHosH2jNOEsGk53fWj43VfRCXqdjiPUwt75GJjkzxUqvsMTttaX9Z9DNXAIpJ+oXuXbPzZcsOsJMsu6TWwfPx3sf2ILbaZz4IHAob5lB+Gw/CfTJKS40heTzkpkurk+hw/4M6L/8FUlTLNZi22d6KdPfWwxaS6vaqDzW29clovNDKfx/+hIP46wk4ewdUpShu3T0RlVB8=
+  on:
+    tags: true
```

Comment:
What will the tag be that it creates? And where does it come from?

Reply:
Ah, I think I understand, so will it publish to npm when we do `git push --tags`?